news.xml

```diff
@@ -16,6 +16,17 @@
     </description>
   </item>
 
+  <item>
+    <pubDate>Mon May 21 2018 00:00:00 GMT</pubDate>
+    <title>
+      Nix-devel mailing list moved
```

Comment:
The subject should be something like "NixOS Discourse forum" since we didn't "move" the mailing list.

Reply:
Thanks, fixed